packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -382,6 +396,25 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     [pinnedParticipantsState, setPinnedParticipantsState, onUnpinParticipantHandler]
   );
 
+  /* @conditional-compile-remove(pinned-participants) */
```

Comment:
nit: To cut down on CC I don't think that we need to CC this. All of the announcer stuff is in the stable API so we should be able to leave this here. Though it will cause build errors since the toggle function wont be used. this can be fixed by moving the CC a layer into  the internal _RemoteVideoTile. wont block on this though if you don't want to change this.